src/modules/map/components/openlayers/utils/print-area-renderer.composable.js

```diff
@@ -40,6 +54,62 @@ function createWorldPolygon() {
     return vectorLayer
 }
 
+function encodeGraticule(dpi, projection) {
+    let gridLayer = 'org.epsg.grid_4326'
+    if (projection.value instanceof CustomCoordinateSystem) {
```

Comment:
because you pass the state as projection and not a vue ref, you don't need the `.value` which is not causing the bug that I mentioned were the grid is in WGS84 instead of lv95

Reply:
ah yes, thanks. I didn't realized the grid is switched.